addons/actions/README.md

```diff
@@ -121,15 +135,15 @@ You can define action handles in a declarative way using `withActions` decorator
 Keys have `'<eventName> <selector>'` format, e.g. `'click .btn'`. Selector is optional. This can be used with any framework but is especially useful for `@storybook/html`.
 
 ```js
-import { storiesOf } from '@storybook/html';
 import { withActions } from '@storybook/addon-actions';
+import Button from './button';
+
+export default {
+  title: 'Button',
+  decorators: [withActions('mouseover', 'click .btn')]
+};
 
-storiesOf('button', module)
-  // Log mouseovers on entire story and clicks on .btn
-  .addDecorator(withActions('mouseover', 'click .btn'))
-  .add('with actions', () => `
-    <div>
-      Clicks on this button will be logged: <button className="btn" type="button">Button</button>
-    </div>
-  `);
+export const fromObject = () => (
```

Comment:
what does `fromObject` mean?

Reply:
It's just the name of the story? Nothing significant at all.

Reply:
how about `defaultView` or something more generic? `fromObject` sounds like it means something but has nothing to do with the story or the example